openapi/user/associate-task-registration.yaml

```diff
@@ -0,0 +1,40 @@
+post:
+  tags:
+    - "TimeRegistration"
+  description: |
+    Associates a time registration - that currently does not have any task associated - to the specified task.
+    Also accepts tags that may be required to make a valid time registration for the specified task.
+  operationId: "associateTimeRegistrationWithTask"
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: "../schemas/user/TimeRegistrationAssociateTaskRequest.yaml"
+  responses:
+    "200":
+      description: "associated"
+      content:
+        application/json:
+          schema:
+            $ref: "../schemas/DefaultUpdateResponse.yaml"
+          examples:
+            success-associated:
+              value: {
+                "id": 12345,
+                "status": "The time registration was successfully associated with task"
+              }
+    "400":
+      description: "Bad associate request"
```

Comment:
Maybe add "For example if the time registration was already associated to a task".